assimilation_code/modules/utilities/ensemble_manager_mod.f90

```diff
@@ -712,6 +712,17 @@ subroutine set_up_ens_distribution(ens_handle)
 type (ensemble_type),  intent(inout)  :: ens_handle
 
 integer :: num_per_pe_below, num_left_over, i
+integer(i8) :: per_pe, suggest_pes
+
+! Check that there are enough pes for the state
+per_pe = ens_handle%num_vars / num_pes
```

Comment:
this looks good.  great idea to suggest a bottom threshold for how many PEs/tasks are needed to make each task's share of the data small enough to fit in an i4.   they'll probably have to add more tasks to fit on a node's memory but this gives them at least a minimum.